resources/views/dashboard/individual.blade.php

```diff
@@ -22,9 +24,10 @@
             </li>
         @endif
         @can('viewAny', App\Models\Project::class)
-            @if ($user->individual->isParticipant())
+            @if ($user->individual->isParticipant() || $user->individual->inProgressParticipatingProjects)
```

Comment:
Likely should be the following:
```suggestion
            @if ($user->individual->isParticipant() || $user->individual->inProgressParticipatingProjects()->count())
```